### PR TITLE
April 2024 RELEASING.md updates

### PR DESCRIPTION
**Pull request by sxa**
Related: https://github.com/adoptium/temurin/pull/41
Draft until the retrospective has been completed. This contains updates while it's fresh in my mind by other changes will be added pending the outcome from [the retrospective](https://github.com/adoptium/temurin-build/pull/3783)

### CODE REVIEW

**Comment by sxa** [comment]
Noting that this does NOT include updates for what we're going to do relating to [tags for dry-run builds](https://github.com/adoptium/temurin/issues/28#issuecomment-2041364554) which will be covered later once further discussions have occurred.

**Review by karianna — APPROVED** [review]

**Comment by karianna** [comment]
@sxa linter failures

**Review by andrew-m-leonard — APPROVED** [review]

**Comment by sxa** [comment]
I'll hold off on merging this for a while in case others want to look over the contents

File: RELEASING.md [hunk 1]
```diff
@@ -54,15 +54,15 @@ Create release branch in the format `vYYYY.MM.NN` on each of the following repos
...
+These branches should be named according to the following format (vYYYY.MM.NN) ,e.g v2023.03.01 , whereby the final element is an incremental counter appended to the year and month of the release.
 
 If anything needs to be merged into the new branch, it should typically be merged into master, then a `git cherry-pick` operation should be done to create a new PR against the release branch. This can typically be merged without further approval.
 
 #### Code Freeze message
 
 Paste the below message into the #release channel in Slack:
 
-With under a week to go until releases, we are entering a lockdown period for the following repositories: temurin-build, ci-jenkins-pipelines, github-release-scripts, containers, installer, and mirror-scripts.
+With under a week to go until releases, we are entering a lockdown period for the `master` branches in the following repositories:  github-release-scripts, containers, installer, and mirror-scripts. The temurin-build, ci-jenkins-pipelines and jenkins-helper master branches are still open for work, however the release branch will be under code-freeze management.
```
Comment (karianna) [inline RELEASING.md:65]:
```suggestion
With under a week to go until releases, we are entering a lockdown period for the `master` branches in the following repositories: github-release-scripts, containers, installer, and mirror-scripts. The temurin-build, ci-jenkins-pipelines and jenkins-helper master branches are still open for work, however the release branch will be under code-freeze management.
```

**Review by karianna — APPROVED** [review]

**Comment by karianna** [comment]
@sxa please do go ahead and merge - folks can always add more later :-)

**Pull request merged**